AISKU/README.md

```diff
@@ -152,15 +155,33 @@ Most configuration fields are named such that they can be defaulted to falsey. A
 
 ## Examples
 
-For runnable examples, see our [Application Insights Demos Page](../Examples/)
+For runnable examples, see [Application Insights Javascript SDK samples](https://github.com/Azure-Samples?utf8=%E2%9C%93&q=application+insights+sdk&type=&language=)
 
 ## Application Insights Web Basic
 
 For a lightweight experience, you can instead install the basic version of Application Insights
 ```
 npm i --save @microsoft/applicationinsights-web-basic
 ```
-This version comes with the bare minimum amount of features and functionalities and relies on you to build it up as you see fit. For example, it performs no auto-collection (uncaught exceptions, ajax, etc). The APIs to send certain telemetry types, like `trackTrace`, `trackEvent`, etc, are not included in this version, so you will need to provide your own wrapper.
+This version comes with the bare minimum amount of features and functionalities and relies on you to build it up as you see fit. For example, it performs no auto-collection (uncaught exceptions, ajax, etc). The APIs to send certain telemetry types, like `trackTrace`, `trackException`, etc, are not included in this version, so you will need to provide your own wrapper. The only api that is available is `track`.
+
+
+## Application Insights backward compatibility validation
```

Comment:
This can just be Backwards Compatibility? or maybe "Upgrading to New Version"

Reply:
Good idea. done.